src/renderer/wg_engine/tvgWgRenderData.cpp

```diff
@@ -298,10 +298,20 @@ void WgRenderDataShape::updateMeshes(WgContext &context, const RenderShape &rsha
         } else if (cmd == PathCommand::Close) {
             polyline.close();
         } else if (cmd == PathCommand::CubicTo) {
+            assert(polyline.pts.count > 0);
+            WgPoint pt0 = polyline.pts.last().trans(rt->m);
+            WgPoint pt3 = WgPoint(rshape.path.pts[pntIndex + 2]).trans(rt->m);
+            size_t nsegs = (uint32_t)(pt0.dist(pt3));
+            if (nsegs <= 1) {
```

Comment:
I'd use the approach with using the length p0-p4 in all cases. just in cxase

Reply:
Fixed